cmd/neofs-cli/modules/object/delete.go

```diff
@@ -28,53 +28,67 @@ func initObjectDeleteCmd() {
 	flags := objectDelCmd.Flags()
 
 	flags.String(commonflags.CIDFlag, "", commonflags.CIDFlagUsage)
-	flags.String(commonflags.OIDFlag, "", commonflags.OIDFlagUsage)
+	flags.StringSlice(commonflags.OIDFlag, nil, commonflags.OIDFlagUsage)
 	flags.Bool(binaryFlag, false, "Deserialize object structure from given file.")
 	flags.String(fileFlag, "", "File with object payload")
+
+	_ = objectDelCmd.MarkFlagRequired(commonflags.CIDFlag)
+	_ = objectDelCmd.MarkFlagRequired(commonflags.OIDFlag)
 }
 
 func deleteObject(cmd *cobra.Command, _ []string) {
 	ctx, cancel := commonflags.GetCommandContext(cmd)
 	defer cancel()
 
 	var cnr cid.ID
-	var obj oid.ID
-	var objAddr oid.Address
+	var objs []oid.ID
+	var objAddrs []oid.Address
 
 	binary, _ := cmd.Flags().GetBool(binaryFlag)
 	if binary {
 		filename, _ := cmd.Flags().GetString(fileFlag)
 		if filename == "" {
 			common.ExitOnErr(cmd, "", fmt.Errorf("required flag \"%s\" not set", fileFlag))
 		}
-		objAddr = readObjectAddressBin(cmd, &cnr, &obj, filename)
+		var obj oid.ID
```

Comment:
create `objs` first and no temp var needed

Reply:
not sure i understand you. it is not always more than one object (if `binary` is provided) and we do not always read `GetStringSlice(commonflags.OIDFlag)` and know its size in advance, only in the first `if` branch. you mean creating a nonempty `objs` slice and just changing it? i do not like this approach usually, it seems error-prone to me, i prefer always to `append`

if you mean to create common vars for all branched, then it will be 
```go
	var cnr cid.ID
	var obj oid.ID
	var objs []oid.ID
	var addr oid.Address
	var objAddrs []oid.Address
```

confusing to me too, creating a var in a place where it is used is a better option for me